### DISCUSSION TITLE
Independent verification of Compute add-on usage metering?

### DISCUSSION THREAD

**Discussion by discretepackets** [supabase/supabase#20018] [Questions]
Hello, has anyone performed independent verification that Supabase is metering Compute and other add-ons correctly?

I have been testing the database for my personal project recently and I am very concerned about how usage is measured.

I ensured that my Compute is Small, and I verified that the web interface shows Small correctly, but I'm being metered for Large currently. The issue is still on-going.

Lack of Audit Logs for more than 24 hours for the Pro plan is also concerning because it prevents historical verification of account activities. Logs for account activities that incur charges such as changes to Compute sizes should be retained and presented for at least the entire billing period plus an additional 30 days so that users can independently verify that Supabase is measuring usage correctly.

**Comment by ventoadmin** [comment]
Interested in this as well. We recently moved up to a Pro plan and our compute has gone from 7hrs a day to 48 hours (in 1 day). No changes in our code, users etc. We're trying to perform an audit of our usage but not 100% all the info is there. Has anyone done this?

**Reply by pierroo** [reply]
any news on this? :/

**Reply by discretepackets** [reply]
I did not receive satisfactory responses from support and I ended up "solving" my issue by not using Supabase entirely.
